src/Command/Serve.php

```diff
@@ -69,9 +85,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         /** @var string $env */
         $env = $input->getOption('env');
 
-        $documentRoot = getcwd() . '/' . $docroot; // TODO: can we do it better?
+        $documentRoot = $this->aliases->get('@root/' . $docroot);
```

Comment:
Alias can not contain `@root` . Can we do without aliases and don't add new dependency?